src/platforms/zulip/triagebot.md

```diff
@@ -0,0 +1,257 @@
+# Triagebot
+
+Triaging on the rust-lang repository is an important step to take care of issues. The triagebot is the tool that allows *anyone* to help by assigning, self-assigning or labeling issues without being a member of the rust-lang organization.
+
+To enable triagebot on a particular repository (currently only in the rust-lang organization), add a `triagebot.toml` file in the repository root. It should have a section per "feature". Please read this page to learn how to enable each feature and the options supported; if you spot something missing please let us know [by filing an issue](https://github.com/rust-lang/rust-forge/issues), thanks!
+
+- [Issue assignment](#issue-assignment)
+- [Issue notifications](#issue-notifications)
+- [Ping a team](#ping-a-team)
+- [Glacier](#glacier)
+- [Triage](#triage)
+- [Apply labels to issues](#apply-labels-to-issues)
+- [Request prioritization](#request-prioritization)
+- [Autolabel an issue](#autolabel-an-issue)
+- [Notify Zulip](#notify-zulip)
+- [Major Changes](#major-changes)
+
+## Issue assignment
+
+Any user belonging to the rust-lang organization can claim an issue via `@rustbot claim`. If the user is not part of the rust-lang organization `rustbot` will choose an assignee and add a "claimed" message in the top-level comment, to signal who the current assignee is. It is possible to override someone else's claim (no warning/error is given).
```

Comment:
rustbot will assign itself, not choose an assignee

Reply:
ok, pls check if now the sentence is correct